contracts/profit/src/msg.rs

```diff
@@ -9,6 +9,7 @@ use sdk::{
 pub struct InstantiateMsg {
     pub cadence_hours: u16,
     pub treasury: Addr,
+    pub oracle: Addr,
```

Comment:
have you updated the initial message in the script?

Reply:
Good catch! Updated in bb9914ef0bf8a0823c3a5b6bb430ca13838be4c2